rusk-recovery/src/bin/state.rs

```diff
@@ -37,7 +41,7 @@ struct Cli {
 fn main() -> Result<(), Box<dyn std::error::Error>> {
     let args = Cli::parse();
     task::run(
-        || rusk_recovery_tools::state::exec(args.build),
+        || rusk_recovery_tools::state::exec(args.build, args.force),
```

Comment:
I'm not a big fan of having method that takes multiple boolean, since it's a signature hard to read (e.g. `rusk_recovery_tools::state::exec(true, false)`)

We might want to have two different workflow here, one for state that are actually build, and one for state that are just deployed from the remote one.

Reply:
I'll try and make a refactor that gets rid of the ambiguous signature.